lib/Models/NewsletterLink.php

```diff
@@ -10,6 +10,7 @@
  */
 class NewsletterLink extends Model {
   public static $_table = MP_NEWSLETTER_LINKS_TABLE;
+  const UNSUBSCRIBE_LINK_SHORT_CODE = '[link:subscription_unsubscribe_url]';
```

Comment:
If you introduced this constant should we also replace strings to start using it?